game/static/game/css/landing.css

```
@@ -606,1637 +622,520 @@ body {
-    font-weight: 800;
-    color: var(--text-secondary);
+
+.footer-socials a:hover {
+    color: var(--accent-gold);
+    background: rgba(240, 192, 64, 0.08);
+    border-color: rgba(240, 192, 64, 0.3);
+    transform: translateY(-3px);
 }
-.pve-mode {
+
+/*BACK TO TOP BUTTON*/
```

Stylelint (17.13.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)